content/clients/_index.md

```diff
@@ -94,47 +93,45 @@ justify = "center"
         <img src="/images/clients/androidtv.svg"/>
         <div class="bannerbox">
             <p class="banner">Android TV</p>
-            <p class="subbanner">Official</p>
         </div>
         <a href="https://play.google.com/store/apps/details?id=org.jellyfin.androidtv" class="button button__accent button__card">Go to Play Store</a>
     </div>
     <div class="card">
         <img src="/images/clients/firetv.svg"/>
         <div class="bannerbox">
             <p class="banner">Fire TV</p>
-            <p class="subbanner">Official</p>
         </div>
         <a href="https://www.amazon.com/gp/aw/d/B07TX7Z725" class="button button__accent button__card">Go to Amazon</a>
     </div>
+    <div class="card">
```

Comment:
We should probably add AirPlay as well, since people will see Chromecase and not seeing AirPlay might make them think that we don't support it.

Reply:
@MrTimscampi Good point! Done!